### ISSUE TITLE
Change 'path' in AD_Acquire According to StrReg Value?

### ISSUE THREAD

**Issue by qzhang234** [aps-8id-dys/ipython-8idiuser#198]
@prjemian I was looking at AD_Acquire and realized that this code, especially plans that call this code, can be made much simpler if `AD_Acquire` is smart enough to get the run directory (e.g. `/home/8-id-i/2020-2/qzhang202006`) from some StrReg. 

@sureshnaps Do we currently have a StrReg for storing the folder name associated with the cycle? If not, do we want to create one?

In a way, this issue is also tied to #180 because fundamentally we want `AD_Acquire` to grab values from StrReg (In the case of #180 it's the name of the workflow for analysis and transfer. How do we do this?

**Comment by prjemian** [comment]
This is also related to https://github.com/BCDA-APS/apstools/issues/333 which provides the APS cycle number from the ESAF database through the [`dm`](https://anaconda.org/aps-anl-tag/aps-dm-api) data management system.

Rather than do something temporary that will soon need to be refactored, let's wait until the *apstools* package is [released](https://github.com/BCDA-APS/apstools/releases) with this new infrastructure.  The content will be provided into an EPICS database local to the beamline.  Features of this system have been requested from several beam lines over the past years. The [milestone](https://github.com/BCDA-APS/apstools/milestone/26) is targeted for completion by 2020-07-15. (That's next week!)

tl;dr: Don't assign a StrReg for the APS cycle number.  Instead, wait for the apstools package to provide it really, really soon.
